pkg/util/helper/taint_test.go

```diff
@@ -357,3 +359,209 @@ func TestGetNoExecuteTaints(t *testing.T) {
 		})
 	}
 }
+
+func TestGetMinTolerationTime(t *testing.T) {
+	now := metav1.Now()
+	later := metav1.NewTime(time.Now().Add(5000))
+	type args struct {
+		noExecuteTaints []corev1.Taint
+		usedTolerations []corev1.Toleration
+	}
+	tests := []struct {
+		name string
+		args args
+		want time.Duration
+	}{
+		{
+			name: "test no noExecute taints",
```

Comment:
How about changing the test case name to `the length of taints is 0`? Same as the other case names.

Reply:
ok